tests/data/_IsmrmrdRawTestData.py

```diff
@@ -99,13 +99,24 @@ def __init__(
         nkx = self.oversampling * nx
         nky = ny
 
-        # Create Cartesian grid for k-space locations
-        ky_idx = self._calc_phase_encoding_steps(nky, self.acceleration, self.sampling_order)
-        kx_idx = range(-nkx // 2, nkx // 2)
-        [kx, ky] = np.meshgrid(kx_idx, ky_idx)
+        if trajectory_type == 'cartesian':
```

Comment:
maybe consider replacing the "_calc_phase_encoding_steps" function by
a "_cartesian_trejectory" and a "_radial_trajectory", both returning ky, kx,pe_line_idx ?